include/cucumber-cpp/internal/hook/HookRegistrar.hpp

```diff
@@ -35,7 +35,7 @@ class BeforeHook : public Hook {
 
 class AroundStepHook : public Hook {
 public:
-    virtual void invokeHook(Scenario *scenario, CallableStep *step);
+    virtual void invokeHookWithStep(Scenario *scenario, CallableStep *step);
```

Comment:
why this name change? there is only one invokeHook and I don't see the point of renaming it